Tasks/ExtractFiles/task.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "5e1e3830-fbfb-11e5-aab1-090c92bc4988",
+  "name": "ExtractFiles",
+  "friendlyName": "Extract Files",
+  "description": "Extract files from a variety of archive types, e.g. .zip, .tar, .7z",
```

Comment:
To call out more of the capabilities, can we change the description to this?
"Extract a variety of archive and compression files such as .7z, .rar, .tar.gz, and .zip."